static/js/sefaria/sefaria.js

```diff
@@ -2144,11 +2193,13 @@ _media: {},
           return x;
       })
   },
-  sectionString: function(ref) {
```

Comment:
data fetching (enable function to be used wihtout cache but with passed data. making it always do so would be better, for util function should not get data for itself)